files/prism/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "prismjs",
+  "repo": "PrismJS/prism",
+  "files": {
+    "include": ["prism.js", "components/**/*", "plugins/**/*", "themes/**/*"]
```

Comment:
This should work
`./components/*.min.js`
`./plugins/**/*.min.js`
`./themes/*.css`